### PR TITLE
executor: add metric for statement counter

### PR DESCRIPTION
**Pull request by tiancaiamao**
With this metric we can knows something like read / write ratio

### CODE REVIEW

File: executor/compiler.go [hunk 1]
```diff
@@ -32,6 +34,14 @@ type Compiler struct {
 // a plan, and we wrap the plan in an adapter as stmt.Statement.
 // If it is not supported, the node will be converted to old statement.
 func (c *Compiler) Compile(ctx context.Context, node ast.StmtNode) (ast.Statement, error) {
+	var stmtName string
+	if stringer, ok := node.(fmt.Stringer); ok {
```
Comment (coocood) [inline executor/compiler.go]: We may use `String` method to print the detail of the Statement later, so I think we should not use `String` as label here, we can define a function that accepts `ast.StmtNode` and returns a label string.
And we don't need to label DDL statement, only frequently used statements like CURD need to be labeled, for other statements, we can just return 'others' label.

Comment (tiancaiamao) [inline executor/compiler.go]: okay...I'll remove `String` method.
BTW, print the detail I think we can use `GoString`, which is called by fmt.Printf "%#v", `String` should be brief.

File: executor/compiler.go [hunk 2]
```diff
@@ -29,18 +27,32 @@ import (
...
+		return "Delete"
+	case *ast.InsertStmt:
+		return "Insert"
+	case *ast.UnionStmt:
+		return "Union"
+	case *ast.UpdateStmt:
+		return "Update"
+	case *ast.CreateIndexStmt:
+		return "CreateIndex"
+	case *ast.CreateTableStmt:
```
Comment (shenli) [inline executor/compiler.go]: CreateDatabase, Replace, DropDatabase and DropTable are also frequently used.

**Comment by shenli** [comment]
LGTM
@coocood PTAL

**Comment by tiancaiamao** [comment]
PTAL @shenli  @coocood

**Comment by coocood** [comment]
LGTM

**Pull request merged**